go.mod

```diff
@@ -1,42 +1,44 @@
 module github.com/smartcontractkit/mcms
 
-go 1.24.4
-
-toolchain go1.24.7
+go 1.25.3
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 require (
 	github.com/aptos-labs/aptos-go-sdk v1.11.0
 	github.com/block-vision/sui-go-sdk v1.1.2
-	github.com/ethereum/go-ethereum v1.15.7
+	github.com/ethereum/go-ethereum v1.16.2
 	github.com/gagliardetto/binary v0.8.0
 	github.com/gagliardetto/solana-go v1.13.0
 	github.com/go-playground/validator/v10 v10.28.0
 	github.com/google/go-cmp v0.7.0
 	github.com/joho/godotenv v1.5.1
 	github.com/karalabe/hid v1.0.1-0.20240306101548-573246063e52
-	github.com/smartcontractkit/chain-selectors v1.0.71
+	github.com/smartcontractkit/chain-selectors v1.0.72
 	github.com/smartcontractkit/chainlink-aptos v0.0.0-20251024142440-51f2ad2652a2
 	github.com/smartcontractkit/chainlink-ccip/chains/solana v0.0.0-20250805210128-7f8a0f403c3a
 	github.com/smartcontractkit/chainlink-ccip/chains/solana/gobindings v0.0.0-20250805210128-7f8a0f403c3a
 	github.com/smartcontractkit/chainlink-sui v0.0.0-20251103204108-d181d6769bab
 	github.com/smartcontractkit/chainlink-testing-framework/framework v0.11.2
-	github.com/smartcontractkit/freeport v0.1.2
-	github.com/spf13/cast v1.7.1
+	github.com/smartcontractkit/chainlink-ton v0.0.0-20251102125826-96306f144e3c
+	github.com/smartcontractkit/freeport v0.1.3-0.20250716200817-cb5dfd0e369e
+	github.com/spf13/cast v1.10.0
 	github.com/stretchr/testify v1.11.1
+	github.com/xssnick/tonutils-go v1.14.1
 	go.uber.org/zap v1.27.0
 	golang.org/x/crypto v0.43.0
 	golang.org/x/tools v0.37.0
 	gotest.tools/v3 v3.5.2
 )
 
+replace github.com/fbsobreira/gotron-sdk => github.com/smartcontractkit/chainlink-tron/relayer/gotron-sdk v0.0.5-0.20250422175525-b7575d96bd4d
```

Comment:
Can you help me to understand why this replace is needed since none of the changes made here seem related to Tron

Reply:
Now removed, it was not needed. Thanks!